crates/symbolicator-service/src/services/download/gcs.rs

```diff
@@ -42,25 +42,21 @@ impl GcsDownloader {
     /// If the cache contains a valid token, then this token is returned. Otherwise, a new token is
     /// requested from GCS and stored in the cache.
     async fn get_token(&self, source_key: &Arc<GcsSourceKey>) -> CacheEntry<Arc<GcsToken>> {
-        metric!(counter("source.gcs.token.requests") += 1);
+        metric!(counter("source.gcs.token.access") += 1);
 
         let init = Box::pin(async {
+            metric!(counter("source.gcs.token.computation") += 1);
             let token = gcs::request_new_token(&self.client, source_key).await;
             token.map(Arc::new).map_err(CacheError::from)
         });
         let replace_if =
             |entry: &CacheEntry<Arc<GcsToken>>| entry.as_ref().map_or(true, |t| t.is_expired());
 
-        let entry = self
-            .token_cache
+        self.token_cache
             .entry_by_ref(source_key)
             .or_insert_with_if(init, replace_if)
-            .await;
-
-        if !entry.is_fresh() {
-            metric!(counter("source.gcs.token.cached") += 1);
```

Comment:
Is this metric gone because it's just the difference between `access` and `computation`?

Reply:
Yes exactly